aoeventtemplates.php

```diff
@@ -447,6 +393,15 @@ function aoeventtemplates_civicrm_validateForm($formName, &$fields, &$files, &$f
       }
     }
   }
+  if ($formName == "CRM_Event_Form_ManageEvent_EventInfo" && !empty($form->getVar('_templateId'))) {
```

Comment:
@Edzelopez I think this needs to be 
```suggestion
  if ($formName == "CRM_Event_Form_ManageEvent_EventInfo" && !empty($fields['template_title'])) {
```

Reply:
Done.